snap/integrity/integrity_test.go

```diff
@@ -227,19 +229,19 @@ case "$1" in
 		exit 0
 		;;
 	format)
-		cp %[1]s %[1]s.verity
-		echo "VERITY header information for %[1]s.verity"
+		truncate -s %[1]d %[2]s.verity
```

Comment:
It looks here like the original copied `snapPath` to `${snapPath}.verity`, but here it seem like you're only operating on `${snapPath}.verity`, which I don't think exists. Could this be the case?

Reply:
`truncate` creates `${snapPath}.verity` of a specific size. I added a comment on the PR explaining why I made this change here: https://github.com/snapcore/snapd/pull/13191#discussion_r1323531502